src/pages/energiatodistus.svelte

```diff
@@ -424,21 +442,23 @@
             class="flex flex-col md:flex-row space-x-2 w-full items-center justify-center">
             <span
               class="w-full md:w-1/2 text-ashblue">{$_('ET_NETTOALA')}:</span>
-            <span class="w-full md:w-1/2">{energiatodistus.nettoala}</span>
+            <span class="w-full md:w-1/2">{energiatodistus[0].lahtotiedot['lammitetty-nettoala']}</span>
           </div>
           <div
             class="flex flex-col md:flex-row space-x-2 w-full items-center justify-center">
             <span
               class="w-full md:w-1/2 text-ashblue">{$_('ET_LAMMITYS_KUVAUS')}:</span>
             <span
-              class="w-full md:w-1/2">{energiatodistus.lammitysjarjestelma}</span>
+              class="w-full md:w-1/2">
+              {$locale == 'sv' ? energiatodistus[0].lahtotiedot.lammitys['lammitysmuoto-1']['kuvaus-sv'] : energiatodistus[0].lahtotiedot.lammitys['lammitysmuoto-1']['kuvaus-fi']}
```

Comment:
Nämä eivät liity käyttöliittymän localeen vaan on osa kaksikielisiä energiatodistuksia, onkohan näitä vielä jotenkin plänätty osaksi käliä. Pitää tutkia.